dietpi/func/dietpi-banner

```diff
@@ -237,6 +238,12 @@ $GREEN_LINE"
 		(( ${aENABLED[1]} == 1 )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[1]} $GREEN_SEPARATOR $(uptime -p 2>&1)"
 		# CPU temp
 		(( ${aENABLED[2]} == 1 )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[2]} $GREEN_SEPARATOR $(print_full_info=1 G_OBTAIN_CPU_TEMP 2>&1)"
+		# RAM usage
+		if (( ${aENABLED[17]} == 1 ))
+		then
+			ramusage=$(free -b | mawk 'NR==2 {CONVFMT="%.0f"; print $3/1024^2" MiB of "$2/1024^2" MiB ("$3/$2*100"%)"}')
+			echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[17]} $GREEN_SEPARATOR $ramusage"
+		fi
```

Comment:
Just since I am no fans of defining variables when they are used a single time only 🙂:
```suggestion
		(( ${aENABLED[17]} == 1 )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[17]} $GREEN_SEPARATOR $(free -b | mawk 'NR==2 {CONVFMT="%.0f"; print $3/1024^2" of "$2/1024^2" MiB ("$3/$2*100"%)"}')"
```
Otherwise